examples/models/llama2/export_llama_lib.py

```diff
@@ -128,6 +129,10 @@ def build_args_parser() -> argparse.ArgumentParser:
             "qnn_8a8w",
             "qnn_16a16w",
             "qnn_16a4w",
+            "coreml",
+            "coreml_qc4",
```

Comment:
Maybe `coreml_qc4w` and `coreml_baseline_qc4w` to make it a bit more clear

Reply:
Renamed to `coreml_qc4w` and `coreml_baseline_qc4w` as suggested